x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/pick_fields.tsx

```diff
@@ -32,10 +33,16 @@ export const PickFieldsStep: FC<StepProps> = ({ setCurrentStep, isCurrentStep })
   const { jobCreator, jobValidator, jobValidatorUpdated } = useContext(JobCreatorContext);
   const [nextActive, setNextActive] = useState(false);
   const [selectionValid, setSelectionValid] = useState(false);
+  const { setIsFlyoutVisible, setFieldName } = useContext(MLJobWizardFieldStatsFlyoutContext);
```

Comment:
this could be wrapped in a `useFieldStatsFlyoutContext` function and exported from `field_stats_flyout`

Similar to:
https://github.com/qn895/kibana/blob/2ab4d991e26d302b6865b78967aef8d060957c48/x-pack/plugins/ml/public/application/contexts/ml/ml_notifications_context.tsx#L96

Reply:
Updated here [`c7bcb4a` (#147322)](https://github.com/elastic/kibana/pull/147322/commits/c7bcb4aba93ef6e0b33edb1517a89c1d14bcbd46)